pkg/docker/config/config.go

```diff
@@ -719,3 +748,15 @@ func normalizeRegistry(registry string) string {
 	}
 	return normalized
 }
+
+// validateKey verifies that the input key does not have any not allowed prefix
+// and returns an indicator if the key is namespaced.
+func validateKey(key string) (isNamespaced bool, err error) {
+	if strings.HasPrefix(key, "http://") || strings.HasPrefix(key, "https://") {
+		return isNamespaced, errors.Errorf("key %s contains http[s]:// prefix", key)
+	}
+
+	// check if the provided key contains one or more subpaths.
+	stripped := stripScheme(key)
```

Comment:
This must do nothing after the rejection above.

Reply:
Opps, yes I removed that.